packages/@react-native-windows/codegen/package.json

```diff
@@ -69,4 +69,4 @@
   "engines": {
     "node": ">= 14"
   }
-}
+}
```

Comment:
I had feedback on one of my PR's. that some unix tools are unhappy when there is no blank line at the end of a file...
I discovered that this was an auto-fixup we have configured in VsCode that runs on save...
I filed #11001  to fix this...
You'll have to either use notepad and add them back or disable 'format on save' temporary in VsCode settings file...

Reply:
Good to know! Thanks for catching that :)